tasks/create-internal-request/README.md

```diff
@@ -8,13 +8,21 @@ Creates an InternalRequest resource to call IIB service
 |------|-------------|----------|---------------|
 | pipelineRunName | The name of the Parent PipelineRun for this task | No | `ir-$(context.pipelineRun.name)` |
 | request | Internal pipeline request name | No | |
-| params | Internal Request parameters | No | |
+| internalRequestParams | Internal Request parameters | No | |
+| dataPath | Path to the JSON string of the merged data to use in the input workspace | Yes | "data.json" |
 | inputDataFile | Optional file to read data from | Yes | "" |
 | updateGenericResult | Should the task update the genericResult result  | Yes | "false" |
 | requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | 360 |
+| subdirectory | Subdirectory inside the workspace to be used | Yes | "" |
 
 ## Changelog
 
+### changes since 0.8
+- adds parameter `dataPath` to support data JSON file
```

Comment:
Nit: Similar to the commit title, these should be in the form of an order if possible. I know we're not always consistent with this - you can see below "add" and "changes" right next to each other.

So here I would propose "add" and "modify".

Reply:
changed.

Reply:
Thanks.